ratis-netty/src/test/java/org/apache/ratis/netty/MiniRaftClusterWithNetty.java

```diff
@@ -30,12 +31,16 @@
 
 import java.io.IOException;
 
+/**
+ * A {@link MiniRaftCluster} with {{@link SupportedRpcType#NETTY}} and {@link SupportedDataStreamType#NETTY}.
+ */
 public class MiniRaftClusterWithNetty extends MiniRaftCluster.RpcBase {
   public static final Factory<MiniRaftClusterWithNetty> FACTORY
       = new Factory<MiniRaftClusterWithNetty>() {
     @Override
     public MiniRaftClusterWithNetty newCluster(String[] ids, RaftProperties prop) {
       RaftConfigKeys.Rpc.setType(prop, SupportedRpcType.NETTY);
+      RaftConfigKeys.DataStream.setType(prop, SupportedDataStreamType.NETTY);
```

Comment:
@szetszwo Can we define `RaftConfigKeys.DataStream.setType(prop, SupportedDataStreamType.NETTY);` in the following code ? If we define it here, other test unrelated to data stream will also start data stream server.
```
public class TestNettyDataStreamWithNettyCluster extends DataStreamClusterTests<MiniRaftClusterWithNetty>
    implements MiniRaftClusterWithNetty.FactoryGet {
  {
    RaftConfigKeys.DataStream.setType(getProperties(), SupportedDataStreamType.NETTY);
  }
}
```

Reply:
That's is a good point.  Let's add another subclass like MiniRaftClusterWithGrpcAndNettyDataStream.